src/components/done/__snapshots__/Done.test.tsx.snap

```diff
@@ -35,11 +35,19 @@ ShallowWrapper {
         />
         <p
             className="call__text"
-        />
-        <strong>
-            TOWN HALL COMPONENT HERE
-        </strong>
-        <br />
+        >
+            <Trans
```

Comment:
This syntax is necessary to do string interpolation with the `react-i18next` library.